universe/supplycommit/env.go

```diff
@@ -87,6 +87,19 @@ func (s SupplyTrees) FetchOrCreate(treeType SupplySubTree) mssmt.Tree {
 	return tree
 }
 
+// FetchSupplyLeavesResp is the response type for fetching the supply
+// leaves for a given asset specifier.
+type FetchSupplyLeavesResp struct {
```

Comment:
nit: Just call this `SupplyLeaves`? Do we need to know there was a fetch call and this is the response? IMO not, we just need to know what's _in_ the type.